x-pack/plugins/apm/public/components/app/correlations/empty_state_prompt.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiEmptyPrompt, EuiSpacer, EuiText } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+export function CorrelationsEmptyStatePrompt() {
+  return (
+    <>
+      <EuiSpacer size="m" />
+      <EuiEmptyPrompt
+        iconType="minusInCircle"
+        title={
+          <EuiText size="s">
+            <h2>
+              {i18n.translate('xpack.apm.correlations.noCorrelationsTitle', {
+                defaultMessage: 'No significant correlations',
+              })}
+            </h2>
+          </EuiText>
+        }
+        body={
+          <>
+            <EuiText size="s">
+              <FormattedMessage
+                id="xpack.apm.correlations.noCorrelationsTextLine1"
+                defaultMessage="Correlations will only be identified if they have significant impact."
+              />
+            </EuiText>
+            {/* Another EuiText element to enforce a line break */}
+            <EuiText size="s">
+              <FormattedMessage
+                id="xpack.apm.correlations.noCorrelationsTextLine2"
+                defaultMessage="Try selecting another time range or remove any added filter."
+              />
+            </EuiText>
+          </>
```

Comment:
It sounds like you want two paragraphs:

```suggestion
          <EuiText size="s">
            <p>
              {i18n.translate("xpack.apm.correlations.noCorrelationsTextLine1", {
                defaultMessage:
                  "Correlations will only be identified if they have significant impact.",
              })}
            </p>

            <p>
              {i18n.translate("xpack.apm.correlations.noCorrelationsTextLine2", {
                defaultMessage:
                  "Try selecting another time range or remove any added filter.",
              })}
            </p>
          </EuiText>
```

Reply:
Updated here https://github.com/elastic/kibana/pull/109187/commits/82d5457f4696584969059f9ad78a897722507d2c